docs/competency-api/audience.md

```diff
@@ -4,4 +4,46 @@ title: Audience
 nav_order: 2
 parent: Competency API
 ---
-# Audience
\ No newline at end of file
+# Audience
+## Update Audience
+
+### Definition
+```HTTP
+https://api.competency-constructor.clark.center/competencies/:competencyId/audience
+```
+
+## Headers
+```JSON
+{
+    Bearer: Bearer.Example.bearer.token
+}
+```
+
+### Body Parameters
+
+| Name | Description | Type | Example |
+| ----------- | ----------- | ----------- | ----------- |
+| type | The type of audience for the competency | string | The undergraduate student |
+| details | Any details necessary about the audience of the competency | string | who has completed basic networking |
+| userId | The ObjectId of the user | ObjectId as a string | 6112745b84804cf5833aa94c |
```

Comment:
So actually the `userId` field does not need to be sent in the body, this will be auto filled after the user has been authorized to make the request in the `@Authorize()` decorator.